GMusicProxy

```diff
@@ -929,13 +940,12 @@ if __name__ == '__main__':
     if config['keyring_service'] is not None or config['keyring_entry'] is not None:
         config['password'] = getKeyringPassword()
 
-    if config['email'] is None or config['password'] is None or len(config['email']) == 0 or len(config['password']) == 0:
-        logger.error(
-            'Please, specify the credentials of your Google account in the config file or on the command-line.')
-        sys.exit(1)
+    if config['email'] is not None or config['password'] is not None:
+        logger.warn('Email and password based login is deprecated. Please switch to OAuth2. See README for more details.')
```

Comment:
`logger.warn` is also deprecated, use `logger.warning` instead.

Reply:
Changed this one to `logger.warning`. There are still a couple of instances of `logger.warn` in the code base, but that should be a seperate PR I think.